53.md

```diff
@@ -0,0 +1,40 @@
+NIP-53
+======
+
+Nostr-based name registration system
+------------------------------------
+
+`draft` `optional` `author:leandro21b`
+
+## Name:
+
+NostrNameSystem (NNS)
+Kind 50
```

Comment:
Markdown eats line breaks. If you don't want these two lines on one, add a line break.

```suggestion
NostrNameSystem (NNS)

Kind 50
```